tests/test_workflows.py

```diff
@@ -119,4 +119,39 @@ async def event_workflow(c: Context):
         assert s.w_output == "Hello from outside! B A"
 
 
-# TODO: test concurrent
+current_time = None
+rand_value = None
+unique_id = None
```

Comment:
why put these here?

Reply:
It's now working if it's inside function, I already test it